skopt/utils.py

```diff
@@ -327,7 +328,7 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
         base_estimator = GaussianProcessRegressor(
             kernel=cov_amplitude * other_kernel,
-            normalize_y=True, random_state=None, alpha=0.0, noise="gaussian",
+            normalize_y=True, alpha=0.0, noise="gaussian",
```

Comment:
Should we pass here also the sampled random integer as a random state?

Reply:
if you pass `random_state` to `cook_estimator` then it will be passed on to the regressor via `base_estimator.set_params(**kwargs)` a few lines later

Reply:
I see